docs/database/server/server.md

```diff
@@ -262,4 +300,4 @@ roles string
 status string
 timezone string
 update_at number
-username string 
+username string 
```

Comment:
not sure what this change is

Reply:
I'm not sure - could be blank line at the end of the file no longer there? No changes made to `User` here.